pkg/controller/clone-controller.go

```diff
@@ -182,6 +192,28 @@ func (r *CloneReconciler) Reconcile(req reconcile.Request) (reconcile.Result, er
 	return reconcile.Result{}, nil
 }
 
+func (r *CloneReconciler) initCloneSourcePodName(pvc *corev1.PersistentVolumeClaim, log logr.Logger, sourcePod *corev1.Pod) error {
```

Comment:
Something about passing the pod to this function seems a little wonky.  If the pod exists, why call this at all?

Reply:
I mean, if there is a legacy pod, then we want to store its name on PVC so rest of the logic is ok? but yes, it does not look good/obvious

Reply:
ok, analyzed it now, if we already have the pod, then we do not need name from annotation :), if we do not have a pod then we need to create or get name, so everything is cool, and this code is indeed not needed

Reply:
done